Sources/Testing/Traits/SPIAwareTrait.swift

```diff
@@ -14,7 +14,8 @@
 /// This protocol refines ``Trait`` in various ways that require the use of SPI.
 /// Ideally, such requirements will be promoted to API when their design
 /// stabilizes.
-@_spi(Experimental) @_spi(ForToolsIntegrationOnly)
+@available(*, deprecated, message: "The SPIAwareTrait protocol is deprecated and no longer used.")
```

Comment:
Was it intentional that you removed `@_spi(Experimental)`?

FWIW we don't need to deprecate SPI and can just delete it.

Reply:
The `SPI.md` document says that `@_spi(ForToolsIntegrationOnly)` will go through a deprecation period. In retrospect, I don't actually think `SPIAwareTrait` needed to have been added to that SPI group, since tools don't really need it AFAIK, but I was trying to honor that guideline. I'd _like_ to delete it outright.

Reply:
Sorry, we don't need to deprecate _experimental_ SPI.

It was added to that group for the benefit of Xcode (although I don't recall if Xcode ended up using it at any point.)

Reply:
But per the guideline in that document, we _do_ need to deprecate `ForToolsIntegrationOnly` decls, right? This protocol is in both SPI groups.

Reply:
I wrote that document. 🙂 The intent is that the rule about `Experimental` takes precedence. Experimental stuff may be modified or deleted at any time regardless of other SPI groups it belongs to.

Reply:
Let's clarify that intent in the document because I do find it ambiguous. I can do that separately.

In the mean time, I'm pleased to go ahead and remove this stuff outright